cypress/e2e/logInLogOut.cy.ts

```diff
@@ -0,0 +1,21 @@
+import HomePage from '../page-object/pages/HomePage';
+
+const homePage = new HomePage();
+const user = Cypress.env('accounts')['user'];
+const SarahConnor = Cypress.env('accounts')['user']['userName'];
```

Comment:
What does the "SarahConnor" variable mean? It's not clear to me. And why variable starts from the first uppercase letter?